### PR TITLE
feat: added GTIN to BoardType and EdgebandType

### PR DESCRIPTION
**Pull request by aldfhg**
[add](feat: added GTIN to BoardType and EdgebandType)

### CODE REVIEW

File: Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs [hunk 1]
```diff
@@ -149,6 +149,12 @@ public class BoardType : IExtensibleDataObject, IContainsUnitSystemDependentProp
         [JsonProperty(Order = 33)]
         public string? ArticleNumber { get; set; }
 
+        /// <summary>
+        /// Gets or sets the gtin.
+        /// </summary>
+        [JsonProperty(Order = 33)]
```
Comment (DaniSchaber) [inline Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs:155]: 34

Comment (aldfhg) [reply Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs]: 34 was taken, I made it 36. Strangely the EdgebandType does not have this Order at all. Resolved.

Comment (HOMAG-Robin-Katzenberger) [reply Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs]: The order number is just for displaying the Json in an explicit order when accessing it through and HTTP Request so it is only for elegance but not really relevant

**Review by HOMAG-Robin-Katzenberger — APPROVED** [review]

File: Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs (hunk 1 above)
Comment (DaniSchaber) [reply Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs]: Shouldn't we add it for Edgebands?

**Pull request merged**